RealmTasks Apple/RealmTasks macOS/ListViewController.swift

```diff
@@ -349,26 +360,51 @@ final class ListViewController<ListType: ListPresentable where ListType: Object>
         view.window?.makeFirstResponder(cellView.textView)
 
         currentlyEditingCellView = cellView
+
+        beginUIWrite()
     }
 
     private func endEditingCells() {
-        guard editing else {
+        guard
+            let cellView = currentlyEditingCellView,
+            let (_, index) = findItemForCellView(cellView)
+        else {
             return
         }
 
+        var item = list.items[index]
+
+        if cellView.text != item.text || cellView.text.isEmpty {
```

Comment:
I've made this change (and removed the dispatch) and it seems to work as intended